src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -536,6 +536,10 @@ def getJobParameters(self, jobList):
             ad['My.CMS_JobType'] = classad.quote(job['task_type'])
             ad['My.CMS_Type'] = classad.quote(activityToType(job['activity']))
             ad['My.CMS_RequestType'] = classad.quote(job['requestType'])
+            if job.get('physicsTaskType', None):
+                ad['My.CMS_extendedJobType'] = classad.quote(job['physicsTaskType'])
```

Comment:
This would be the final classad meant for monit to export as, as a replacement of the current guessTaskType method used to set the `CMS_TaskType`.
I called it `CMS_extendedJobType` because in case of non-physics task jobs, this will display "LogCollect", "Merge", "Cleanup", "Repack", "Express", which a are non-physics types.
We could just provide just a `CMS_PhysicsTaskType` but then some if conditionals would be required at the monit level to get the general job task types (a combination of `CMS_PhysicsTaskType`. and `CMS_JobType`) and I would rather avoid them doing that.

Reply:
That is a good point! The current logic already mixes up task definition with job type, so if we decide to proceed with the latter, I think it would be fine.
Otherwise, I would rather suggest `CMS_extendedTaskType` then.
Based on your comment, I guess I have no preference now.